docs/user/tutorials/01-10-mtls-authentication/configure-mtls-Gardener-certs.md

```diff
@@ -0,0 +1,568 @@
+
+    ```bash
+    cat <<EOF | kubectl apply -f -
+    apiVersion: cert.gardener.cloud/v1alpha1
+    kind: Certificate
+    metadata:
+      name: domain-certificate
+      namespace: "istio-system"
+    spec:
+      secretName: kyma-mtls
+      commonName: "${GATEWAY_DOMAIN}"
+      issuerRef:
+        name: garden
+    EOF
+    ```
+    To verify that the Secret with Gateway certificates is created, run:
+   
+    ```bash
+    kubectl get secret -n istio-system kyma-mtls
+    ```
+
+4. Prepare the client's certificates.
+
+   To illustrate the process, this procedure uses self-signed client certificates.
+   >[!WARNING]
+   > For production deployments, use trusted certificate authorities to ensure proper security and automatic certificate management.
+
+    1. Create the client's root CA.
+         ```bash
+         CLIENT_ROOT_CA_KEY_FILE="client_root_ca_cn.key"
+         CLIENT_ROOT_CA_CRT_FILE="client_root_ca_cn.crt"
+         openssl req -x509 -sha256 -nodes -days 365 -newkey rsa:2048 -subj "/O=Example Client Root CA ORG/CN=Example Client Root CA CN" -keyout "${CLIENT_ROOT_CA_KEY_FILE}" -out "${CLIENT_ROOT_CA_CRT_FILE}"
+         ```
+
+    2. Create the client's certificate.
+        ```bash
+        CLIENT_CERT_CRT_FILE="client_cert_cn.crt"
+        CLIENT_CERT_CSR_FILE="client_cert_cn.csr"
+        CLIENT_CERT_KEY_FILE="client_cert_cn.key"
+        openssl req -out "${CLIENT_CERT_CSR_FILE}" -newkey rsa:2048 -nodes -keyout "${CLIENT_CERT_KEY_FILE}" -subj "/CN=Example Client Cert CN/O=Example Client Cert Org"
+        ``` 
+
+    3. Sign the client's certificate.
+
+        ```bash
+        openssl x509 -req -days 365 -CA "${CLIENT_ROOT_CA_CRT_FILE}" -CAkey "${CLIENT_ROOT_CA_KEY_FILE}" -set_serial 0 -in "${CLIENT_CERT_CSR_FILE}" -out "${CLIENT_CERT_CRT_FILE}"  
+        ``` 
+
+5.  Create a Secret with Client CA Cert for mTLS Gateway. For more information on the convention that the Secret must use, see [Key Convention](https://istio.io/latest/docs/tasks/traffic-management/ingress/secure-ingress/#key-formats).
+
+    ```bash
+    kubectl create secret generic -n istio-system "kyma-mtls-cacert" --from-file=cacert="${CLIENT_ROOT_CA_CRT_FILE}"
+    ```
+
+6.  Create an mTLS Gateway.
+ 
+    ```bash
+    cat <<EOF | kubectl apply -f -
+    apiVersion: networking.istio.io/v1alpha3
+    kind: Gateway
+    metadata:
+      name: kyma-mtls-gateway
+      namespace: test
+    spec:
+      selector:
+        app: istio-ingressgateway
+        istio: ingressgateway
+      servers:
+        - port:
+            number: 443
+            name: mtls
+            protocol: HTTPS
+          tls:
+            mode: MUTUAL
+            credentialName: kyma-mtls
+          hosts:
+            - "${GATEWAY_DOMAIN}"
+    EOF
+    ```
+
+11. To expose your workload, create an APIRule custom resource.
```

Comment:
this might be step 7, not 11. Not entirely sure, but I believe the md jumps from step 6 to 11 etc.

Reply:
yes, you're right
thank you :)